src/cloud/project/new-relic.md

```diff
@@ -8,6 +8,14 @@ functional_areas:
   - Services
 ---
 
+{{ site.data.var.ece }} subscriptions include access to New Relic services to help manage, monitor, and troubleshoot your applications and infrastructure by collecting, viewing, and analyzing data from your {{ site.data.var.ece }} project.
```

Comment:
I hesitate to use "subscription". New Relic uses "subscription" with a different meaning. I think we use "plan" in the docs, **but** I would use "account" here. When I view New Relic, I have multiple projects. I have to select a Magento Account when I want to see what's going on in my instance.

Reply:
Changed references to projects for consistency and to avoid confusion with New Relic account.